workflows_core/api/api.py

```diff
@@ -494,3 +494,92 @@ def _merge_tags(
             ),
         )
         return get_response(response)
+    
+    @retry()
+    def _bulk_update_keyphrase(
+        self,
+        dataset_id: str,
+        field: str,
+        keyphrase: List,
+        updates: List
```

Comment:
What are `keyphrase` and  `updates` for?
Why here `keyphrase` is List while it is str in other functions?

Reply:
Good catch - `keyphrase` should have been a string. 

`keyphrase` is what Josh means by `alias` here.